src/style.css

```diff
@@ -1 +1,71 @@
 /* Styles go here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+  padding: 0;
+}
```

Comment:
Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser to read your web document